bitsofgood/pencils-for-success
# Add simple map to landing page

### Description

Adds map

**Related Issues/PRs:** 

#93 

### Test Plan

- Open landing page (`/`)
- See map display correctly
